docs/getting_started/install_ez_platform.md

```diff
@@ -122,6 +122,9 @@ After a moment the installer will ask you to provide a few parameters:
     Do not commit the secret to version control systems, or share it with anyone who does not strictly need it.
     If you have any suspicion that the secret may have been exposed, replace it with a new one.
     The same goes for other secrets, like database password, Varnish invalidate token, JWT passphrase, etc.
+    
+    Remember, that after changing the app secret you should regenerate application cache and log out all the users.
```

Comment:
```suggestion
    After changing the app secret, make sure that you clear the application cache and log out all the users.
```

Reply:
Hi @konradoboza , do you want this change to be merged up to newer versions of the documentation, up to master?

Reply:
As mentioned in the PR description, it should be merged into all supported branches taking Symfony doc URL change into account along the way.

Reply:
+1 to this one.